src/utils/intersection-observer-polyfill.js

```diff
@@ -100,8 +101,15 @@ export function getIntersectionChangeEntry(element, owner, hostViewport) {
  * @return {boolean}
  */
 export function nativeIntersectionObserverSupported(win) {
+  if (!('IntersectionObserver' in win)) {
+    return false;
+  }
+  // eslint-disable-next-line no-undef
+  if (INTERSECTION_OBSERVER_POLYFILL || getMode().localDev || getMode().test) {
+    // For the new stub polyfill it's enough to have a stub to be functional.
+    return true;
+  }
```

Comment:
Why is this change needed?

Reply:
Technically, the polyfill doesn't define the prototype for `IntersectionObserverEntry`. E.g. this will fail: ` 'intersectionRatio' in win.IntersectionObserverEntry.prototype`. That being said, post-launch, this whole method and all downstream code will be gone, so I figured I don't need to press this issue.